pyproject.toml

```diff
@@ -7,7 +7,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "geoapis"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
Do you think we could make this 0.3.1 as I don't think this constitutes a major functionality change :)